src/Microsoft.AspNetCore.TestHost/WebSocketClient.cs

```diff
@@ -163,6 +167,11 @@ internal void ServerCleanup(Exception exception)
                 _application.DisposeContext(Context, exception);
             }
 
+            private void RequestCancelled()
```

Comment:
The method doesn't really need to be extracted out here. Inline this with the `cancellationToken.Register(...)` call. That way you also do not need the `_cancellationToken` field.